pkg/controller/machinehealthcheck/machinehealthcheck_controller.go

```diff
@@ -488,7 +488,7 @@ func (r *ReconcileMachineHealthCheck) healthCheckTargets(targets []target, timeo
 			continue
 		}
 
-		if t.Machine.DeletionTimestamp == nil && t.Node != nil {
+		if _, externalRemediation := t.Machine.Annotations[machineExternalAnnotationKey]; t.Machine.DeletionTimestamp == nil && t.Node != nil && !externalRemediation {
```

Comment:
Pretty nitty but, maybe this would be easier to read if this was extracted to a helper, eg
```
func isBeingExternallyRemediated(machine *mapiv1.Machine) bool {
  if machine.Annotations == nil {
    return false
  }
  _, externalRemediation := machine.Annotations[machineExternalAnnotationKey];
  return externalRemediation
}
```
This would make working out what the if statement on this line is doing a bit easier IMO